ew_agents/vertex_ai_integration.py

```
@@ -32,7 +32,7 @@ class VertexAIAgentEngine:
     Provides cloud-based AI capabilities and deployment support.
     """
     
-    def __init__(self, project_id: str = None, location: str = "europe-west1"):
+    def __init__(self, project_id: Optional[str] = None, location: str = "europe-west1"):
```

Ruff (0.12.2)

Undefined name `Optional`

(F821)